01/mh70cz/wordvalue.py

```diff
@@ -0,0 +1,39 @@
+from data import DICTIONARY, LETTER_SCORES
+
+def load_words():
+    """Load dictionary into a list and return list"""
+    dl = []
```

Comment:
You can also use `yield` to make this a generator, less code and performs faster